eda/src/autogluon/eda/visualization/layouts.py

```diff
@@ -7,7 +7,15 @@
 from .base import AbstractVisualization
 from .jupyter import JupyterMixin
 
-__all__ = ["MarkdownSectionComponent", "SimpleVerticalLinearLayout", "SimpleHorizontalLayout", "TabLayout"]
+__all__ = [
+    "MarkdownSectionComponent",
+    "SimpleVerticalLinearLayout",
+    "SimpleHorizontalLayout",
+    "TabLayout",
+    "PropertyRendererComponent",
```

Comment:
If you do a new revision, you could make these alphabetical

Reply:
I'm relying on `isort` and `black` for formatting. If it is passing, I assume it's PEP-compliant.